src/libs/actions/Report.js

```diff
@@ -736,15 +741,21 @@ function subscribeToUserEvents() {
                 {error, pusherChannelName, eventName: Pusher.TYPE.REPORT_TOGGLE_PINNED},
             );
         });
+}
 
+/**
+ * Setup reportComment push notification callbacks.
+ */
+function subscribeToReportCommentPushNotifications() {
     PushNotification.onReceived(PushNotification.TYPE.REPORT_COMMENT, ({reportID, reportAction}) => {
         Log.info('[Report] Handled event sent by Airship', false, {reportID});
         updateReportWithNewAction(reportID, reportAction);
     });
 
     // Open correct report when push notification is clicked
     PushNotification.onSelected(PushNotification.TYPE.REPORT_COMMENT, ({reportID}) => {
-        Navigation.navigate(ROUTES.getReportRoute(reportID));
+        Linking.openURL(`${CONST.DEEPLINK_BASE_URL}${ROUTES.getReportRoute(reportID)}`)
+            .then(() => Navigation.setDidTapNotification());
```

Comment:
NAB, could be wrong but I don't see the reason to wait for this promise to resolve.

Reply:
Yeah I wasn't sure here either. I guess I'll get rid of `.then`

Reply:
Done.